### ISSUE TITLE
Does it have any CHANGELOG?

### ISSUE THREAD

**Issue by rivajunior** [broofa/simplur#1]
I am thinking in starting using this lib, and I think that is very important to a lib have a CHANGELOG for those who use it, always know what is changing when an update comes.

But I was not able to find an history of past versions. Does it have any?

**Comment by broofa** [comment]
Good point.  I'll set this up as part of the next release (whenever that is).
